updates.jenkins.io.tf

```
@@ -0,0 +1,56 @@
+# Storage account
+resource "azurerm_resource_group" "updates_jenkins_io" {
+  name     = "updates-jenkins-io"
+  location = var.location
+  tags     = local.default_tags
+}
+
+resource "azurerm_storage_account" "updates_jenkins_io_files" {
+  name                          = "updatesjenkinsiofiles"
+  resource_group_name           = azurerm_resource_group.updates_jenkins_io.name
+  location                      = azurerm_resource_group.updates_jenkins_io.location
+  account_tier                  = "Standard"
+  account_replication_type      = "GRS"    # recommended for backups
```

```suggestion
  account_replication_type      = "LRS"    # recommended for backups
```

Note sure why you would want to use GRS? https://learn.microsoft.com/en-us/azure/storage/common/storage-redundancy states that LRS are more cost effective and it's not clear why we would want to georeplicate?